app-accessibility/yasr/yasr-0.6.9-r1.ebuild

```diff
@@ -16,16 +16,28 @@ IUSE="nls"
 RDEPEND=""
 DEPEND="nls? ( sys-devel/gettext )"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.6.9-automake113.patch
+	"${FILESDIR}"/${PN}-0.6.9-gettext019.patch
+	"${FILESDIR}"/${PN}-0.6.9-gcc43.patch
+	"${FILESDIR}"/${PN}-0.6.9-remove-m4.patch
+)
+
 src_prepare() {
+	eapply "${PATCHES[@]}"
 	local x=/usr/share/gettext/po/Makefile.in.in
 	[[ -e $x ]] && cp -f $x po/ #330879
 
-	epatch "${FILESDIR}"/${P}-automake113.patch \
-		"${FILESDIR}"/${P}-gettext018.patch \
-		"${FILESDIR}"/${P}-gcc43.patch \
-		"${FILESDIR}"/${P}-remove-m4.patch
-
 	rm -r "${S}"/m4
+
+	sed -i \
+	's:^\(synthesizer=emacspeak server\):#\1:
+	s:^\(synthesizer port=|/usr/local/bin/eflite\):#\1:
+	s:^#\(synthesizer=speech dispatcher\):\1:
+	s:^#\(synthesizer port=127.0.0.1.6560\):\1:' yasr.conf
+
+	mv configure.{in,ac}
+	eapply_user
```

Comment:
eapply_user should be done before doing any changes to files. Replace this and eapply with default.